src/plugins/graph/adornments/movable-line/movable-line-model.ts

```diff
@@ -75,36 +78,57 @@ export const MovableLineInstance = types.model("MovableLineInstance", {
   }
 }));
 
+export interface IMovableLineInstance extends Instance<typeof MovableLineInstance> {}
+
 export const MovableLineModel = AdornmentModel
 .named('MovableLineModel')
 .props({
   type: 'Movable Line',
-  lines: types.map(MovableLineInstance)
+  lines: types.array(MovableLineInstance)
```

Comment:
@bgoldowsky  There's no ordering of lines. I just thought making it an array simplified things a little. I believe the main reason it was a map before was because in CODAP the plot can be split and there can be an instances of the movable line in each subplot. We essentially used a key that matched each instance in the `lines` map to its subplot in the graph. Here in CLUE, I believe it's not possible to have split plots and it's not a feature we'll be adding. Before I changed `lines` to an array, I found that all I was doing was using `String(index)` for the key, and figured that changing the property to an array and using the straight `index` values to get each line made things a bit simpler.

Is there an advantage to using a map that I'm overlooking? Or maybe in the very least it'd be best to continue using map so it doesn't break with CODAP? I don't think we're trying to keep the two graphs in sync anymore, but maybe it'd still be best to not make unnecessary changes?

@kswenson, do you have any thoughts on any of the above?

Reply:
1. I wouldn't be so hasty to assume that CLUE will never want to split plots just because they don't have that need now.
2. Maps are easier to reconcile than arrays so they can lead to better patches when handling actions.

My inclination would be to keep the map for now.

Reply:
OK, I've switched it back to map.